### ISSUE TITLE
setup.py

### ISSUE THREAD

**Issue by firesharker** [nwojke/pymotutils#1]
Hi,
do you plan to add a `setup.py`?
If there is one, you can add this repo to your project's `requirements.txt` or setup script or simply install as a package with `pip install git+git://repo`. That way you do not have to use `git submodule` and create symlinks to the actual package to be able to do `import pymotutils`.

**Comment by nwojke** [comment]
Hi firesharker,

its a great idea. I haven't found the time lately to work on this project, but I think it is a great addition and I will add it as soon as there is more time.

**Comment by nwojke** [comment]
Or, if you like and find the time for it, you could go ahead and submit a pull request.

**Comment by firesharker** [comment]
Back then I decided to leave it to you to write an official version, but I created these files as a test. It's just a slightly modified sample code.

**setup.py**
```
from setuptools import setup, find_packages

from os import path

here = path.abspath(path.dirname(__file__))

# Get the long description from the README file
with open(path.join(here, 'README.md'), encoding='utf-8') as f:
    long_description = f.read()

with open('requirements.txt') as f:
    required = f.read().splitlines()

setup(
    name='pymotutils',
    version='1.0.0',
    description='multiple object tracking utilities',
    long_description=long_description,
    url='https://github.com/nwojke/pymotutils',
    author='Nicolai Wojke',
    author_email='sample@address.com',
    license='GPL3',
    classifiers=[
        # How mature is this project? Common values are
        #   3 - Alpha
        #   4 - Beta
        #   5 - Production/Stable
        'Development Status :: 3 - Alpha',

        # Indicate who your project is intended for
        'Intended Audience :: Master Hackers',
        'Topic :: Computer Vision :: Multiple Object Tracking',

        # Pick your license as you wish (should match "license" above)
        'License :: OSI Approved :: GPL 3.0 License',

        # Specify the Python versions you support here. In particular, ensure
        # that you indicate whether you support Python 2, Python 3 or both.
        'Programming Language :: Python :: 2',
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.3',
        'Programming Language :: Python :: 3.4',
        'Programming Language :: Python :: 3.5',
    ],
    keywords='mot',
    packages=find_packages(exclude=['examples']),
    install_requires=required
)

```
**setup.cfg**
```
[bdist_wheel]
universal=1
```
**requirements.txt**
```
setuptools==36.4.0
numpy==1.13.1
six==1.11.0
scikit_learn==0.19.0
```

**Comment by nwojke** [comment]
Has been addressed in PR #4. Thanks for the code.
